cli/command/manifest/create_list.go

```diff
@@ -0,0 +1,82 @@
+package manifest
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/distribution/reference"
+	"github.com/docker/docker/registry"
+)
+
+type annotateOpts struct {
+	amend bool
+}
+
+func newCreateListCommand(dockerCli command.Cli) *cobra.Command {
+
+	opts := annotateOpts{}
+
+	cmd := &cobra.Command{
+		Use:   "create newRef manifest [manifest...]",
+		Short: "Create a local manifest list for annotating and pushing to a registry",
+		Args:  cli.RequiresMinArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return createManifestList(dockerCli, args, opts)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.BoolVarP(&opts.amend, "amend", "a", false, "Amend an existing manifest list transaction")
```

Comment:
I've just starting reading through and this is the first time I've come across the term transaction in these commands. What does that refer to?

A quick scan (so I might have missed something) suggests this is an internal concept that isn't mentioned to the user anywhere but here. Can we remove this term? Maybe it should be "local manifest list" ?